social_core/backends/azuread_tenant.py

```diff
@@ -2,9 +2,12 @@
 
 from cryptography.hazmat.backends import default_backend
 from cryptography.x509 import load_der_x509_certificate
-from jwt import DecodeError, ExpiredSignatureError
+from jwt import (  # pragma: no cover
```

Comment:
Why the `pragma: no cover` here?

Reply:
Codecov (not my favorite tool) will fail because this line is not covered by a test.

Reply:
There are more uncovered lines here, I don't see that a concern. Please remove that comment.